server/src/main/java/com/objectcomputing/checkins/notifications/email/EmailSender.java

```diff
@@ -1,5 +1,5 @@
 package com.objectcomputing.checkins.notifications.email;
 
 public interface EmailSender {
-    public void sendEmail(String subject, String content);
+    public void sendEmail(String subject, String content, String... recipients);
```

Comment:
should this be String ...recipients or am I reading that wrong?

Reply:
The three dots in java are called Varargs and they work a bit different then the spread operator for JavaScript.  I saw it being used this way in one of the commits in the feedback branch and had to look it up myself (it seemed like a good use though so I repeated it).  Java treats the recipients variable as an array of strings but when you call the method, if you only have one recipient, you can pass the string directly and skip putting it into an array. You can also explicitly pass multiple String recipients and Java will convert it to an array for you. Of course you can also just pass in an array as well if that's what you're working with already. 

So all of the following are valid ways to call the method:
sendEmail (subject, content, "someEmail@gmail.com");
sendEmail (subject, content, "someEmail@gmail.com", "anotherEmail@gmail.com");

String[] emails = new String[] {"someEmail@gmail.com", "anotherEmail@gmail.com", "aThirdEmail@gmail.com"};
sendEmail(subject, content, emails);

Reply:
Nice, I'll have to keep Varargs in mind then